src/convenience/frameworks.ts

```diff
@@ -241,8 +410,8 @@ const http: FrameworkAdapter = (req, res) => {
 };
 
 /** koa web framework */
-const koa: FrameworkAdapter = (ctx) => ({
-    update: Promise.resolve(ctx.request.body),
+const koa: KoaAdapter = (ctx) => ({
+    update: Promise.resolve(JSON.parse(ctx.body)),
```

Comment:
Are these changes actually correct? It seems to me from the docs that `ctx.body` is a response alias, so obtaining the request body should still be done via `ctx.request.body` after applying a `bodyParser`.

```suggestion
const koa: KoaAdapter = (ctx) => ({
    update: Promise.resolve(ctx.request.body),
```

Is there something I'm missing?

Reply:
There doesn't seem to be a body property in the types.

Reply:
Are you referring to using the `bodyparser` plugin?  https://github.com/koajs/bodyparser

Reply:
Yes